honeypotio/staticpages
# Growth: /join Speed Optimizations #2071

How to Test:
- Visit [google page insights](https://developers.google.com/speed/pagespeed/insights/)
- Paste http://staging.honeypot.co/pr-443/lp/join which where this staging server is located
- It should be green

ticket: https://github.com/honeypotio/honeypot/issues/2071
